src/resolve/resolved-dns-dnssec.c

```diff
@@ -445,9 +534,18 @@ static void md_add_uint16(gcry_md_hd_t md, uint16_t v) {
         gcry_md_write(md, &v, sizeof(v));
 }
 
-static void md_add_uint32(gcry_md_hd_t md, uint32_t v) {
+static size_t fwrite_uint8(FILE *fp, uint8_t v) {
+        return fwrite(&v, sizeof(v), 1, fp);
```

Comment:
hmm, thinking about this, these function prototypes are actually weird since they are declared size_t, but will only return 0 or 1... (that's because fwrite() returns the "nmemb" paramater on success, not the first. It's kinda strange to return a "size_t" if we never actually return anything non-binary there... dunno.

Given that we don't care about the return value, and these functions are static anyway, I am pretty sure it would make more senseto just return void here...